packages/terra-functional-testing/src/commands/utils/describeTests.js

```diff
@@ -0,0 +1,38 @@
+const setViewport = require('./setViewport');
+const describeViewports = require('./describeViewports');
+
+/**
+ * Mocha describe block to filter tests based on form factors, locales, and themes.
+ * The block of tests will only execute if the form factor, locale, and theme in the current configuration match what is defined in the options.
+ * If any option is not defined, then the block of tests will not be filtered based on that particular option.
+ * @param {string} title - The describe block title.
+ * @param {Object} options - An object containing arraies of formFactors, locales, and themes that the block of tests will only qualify to execute in.
+ * @param {string} options.formFactors - The form factors that the block of tests only execute in.
+ * @param {string} options.locales -  The language locales that the block of tests only execute in.
+ * @param {string} options.themes - The themes that the block of tests only execute in.
+ * @param {function} - The block of tests to execute against each viewport.
+ */
+const describeTests = (title, options = {}, fn) => {
```

Comment:
Other names we can use are `filterTests`, `filterTestsWithOptions`. I'm thinking we may need to create a separate helper for running all permutations for these options and can then use `describeTests` for that.